code/controllers/subsystem/jobs.dm

```diff
@@ -502,11 +502,9 @@ SUBSYSTEM_DEF(jobs)
 					G.prescription = TRUE
 					G.name = "prescription [G.name]"
 					H.update_nearsighted_effects()
-	return H
-
-
-
 
+	H.create_log(MISC_LOG, "Spawned as \an [H?.dna?.species ? H.dna.species : "Undefined species"] named [H]. [joined_late ? "Joined during the round" : "Roundstart joined"] as job: [rank].")
```

Comment:
```suggestion
	H.create_log(MISC_LOG, "Spawned as \an [H.dna?.species ? H.dna.species : "Undefined species"] named [H]. [joined_late ? "Joined during the round" : "Roundstart joined"] as job: [rank].")
```
don't need to null guard `H` since its null checked up top and this proc will runtime anyway if its null